oidc_provider/lib/endpoints/authorize.py

```diff
@@ -112,25 +140,32 @@ def validate_params(self):
             raise AuthorizeError(self.params['redirect_uri'], 'invalid_scope', self.grant_type)
 
         # Nonce parameter validation.
-        if self.is_authentication and self.grant_type == 'implicit' and not self.params['nonce']:
-            raise AuthorizeError(self.params['redirect_uri'], 'invalid_request', self.grant_type)
+        #if self.is_authentication and self.grant_type == 'implicit' and not self.params['nonce']:
+        #    raise AuthorizeError(self.params['redirect_uri'], 'invalid_request', self.grant_type)
+
+        #if self.grant_type == 'hybrid' and not self.params['nonce']:
+         #   raise AuthorizeError(self.params['redirect_uri'], 'invalid_request', self.grant_type)
+        # check if user is no user and prompt = none
+
+        if self.is_authentication and self.params['prompt'] == 'none' and not self.request.is_authenticated:
+            raise AuthorizeError(self.params["redirect_uri"],'login_required',self.grant_type)
 
         # Response type parameter validation.
         if self.is_authentication \
                 and self.params['response_type'] not in self.client.response_type_values():
-            raise AuthorizeError(self.params['redirect_uri'], 'invalid_request', self.grant_type)
+            raise AuthorizeError(self.params['redirect_uri'], 'in	login_requiredvalid_request', self.grant_type)
```

Comment:
Should be `invalid_request`

Reply:
I moved the PR to an older commit with had support for form_post (PASSED Implicit Form Post Test Suite), acr_values and, Dsicovery passed CONFIG OP